tiledb/sm/storage_manager/context.cc

```diff
@@ -124,6 +125,24 @@ stats::Stats* Context::stats() const {
   return stats_.get();
 }
 
+std::string Context::abs_path(const std::string& path) {
+  if (storage_manager_ != nullptr) {
+    bool found = false;
+    std::string local_root_path =
+        storage_manager_->config().get("vfs.local_root_path", &found);
+    if (found && local_root_path.length() > 0) {
```

Comment:
Not seeing anywhere that some attempt is made to validate that vfs.local_root_path is an a validly formed absolute path prefix, not sure if that should be done or not???

Reply:
In the abs_path function of win.cc and posix.cc, there are some validation tests